ansible/playbooks/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -772,6 +772,7 @@ default_text_search_config = 'pg_catalog.english'
 #include_if_exists = '...'		# include file only if it exists
 #include = '...'			# include file
 include = 'postgresql-epiphany.conf'	# Epiphany managed configuration that overrides settings above
+include_if_exists = 'postgresql-lumada.conf' #Lumada managed configuration that overrides settings above
```

Comment:
It will be good to make this configurable, name wise from the configuration perspective.